src/main/java/org/fundacionjala/coding/KevinHerrera/Movies/Customer/Customer.java

```diff
@@ -0,0 +1,111 @@
+package org.fundacionjala.coding.KevinHerrera.Movies.Customer;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.fundacionjala.coding.KevinHerrera.Movies.Movie.Movie;
+import org.fundacionjala.coding.KevinHerrera.Movies.Rental.RentalMovie;
+import org.fundacionjala.coding.KevinHerrera.Movies.Rental.RentalFactory;
+
+/**
+ * @author KevinHerrera - AWT-[01].
+ * @version 0.1
+ */
+public class Customer implements IRentMovie {
+    private String name;
+    private List<RentalMovie> rentedMoviesList;
+    private int frequentRenterPoints = 0;
+
+    /**
+     * .
+     * @param name name
+     */
+    public Customer(final String name) {
+        this.name = name;
+        rentedMoviesList = new ArrayList<RentalMovie>();
+    }
+
+    /**
+     * .
+     * @return value
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * .
+     * @return value
+     */
+    public String statement() {
+        StringBuilder bld = new StringBuilder();
+        bld.append("RentalMovie Record for: " + getName() + System.lineSeparator());
+        for (RentalMovie rental : rentedMoviesList) {
+
+            //show figures for this rental
+            bld.append(String.format("Title: %s(%s)||\tRent Amount: %d $||\tDays Rented: %s day(s)%n",
+                    rental.getMovie().getTitle(),
+                    rental.getMovie().priceCodeToString(),
+                    (int) rental.getRentAmount(),
+                    rental.getDaysRented())
+            );
+        }
+        bld.append("Amount owned is " + getTotalBillAmount()  + "$" + System.lineSeparator());
+        bld.append("You earned:" + frequentRenterPoints + " frequent renter points");
+        String result = bld.toString();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)